src/.storybook/config.ts

```diff
@@ -0,0 +1,9 @@
+import { configure } from "@storybook/react"
+
+const req = require.context("../../../../src", true, /stories\.(tsx)$/)
```

Comment:
I'm no `path` master (yet)—there is likely a better way to do this, but `context` needed static path... I guess 🤷‍♂